marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx

```diff
@@ -178,11 +180,30 @@ describe("Group Nodes", () => {
 
       //Validate tooltip appears
       canvas.click(orderCoordinates.x, orderCoordinates.y, {force: true});
-      graphExplore.getTooltip().should("contain", "Group of Product records");
-      graphExplore.getTooltip().should("contain", "Click to expand 3 sample records in this group.");
-      graphExplore.getTooltip().should("contain", "Double click to expand all records in this group.");
+      graphExplore.getTooltip().should("contain", "Group of Product records")
+      .should("contain", "Click to expand 3 sample records in this group.")
+      .should("contain", "Double click to expand all records in this group.");
     });
 
   });
 
+  it("Validate Expand records on table", () => {
+    cy.wait(1000)
+    cy.log("**Right click and select expand in the table**");
+    graphExplore.focusNode(ExploreGraphNodes.PRODUCT_GROUP);
+    graphExplore.getPositionsOfNodes(ExploreGraphNodes.PRODUCT_GROUP).then((nodePositions: any) => {
+      let orderCoordinates: any = nodePositions[ExploreGraphNodes.PRODUCT_GROUP];
+      const canvas = graphExplore.getGraphVisCanvas();
+     
+      // Right click and expand 3 records of the node
```

Comment:
Can you update the comment here ? It still says expand 3 records of the node.

Reply:
Good catch, thanks!